subcmds/report.go

```diff
@@ -95,8 +96,11 @@ func (p *ReportCmd) SetFlags(f *flag.FlagSet) {
 	f.Float64Var(&c.Conf.CvssScoreOver, "cvss-over", 0,
 		"-cvss-over=6.5 means reporting CVSS Score 6.5 and over (default: 0 (means report all))")
 
-	f.BoolVar(&c.Conf.Diff, "diff", false,
-		"Difference between previous result and current result")
+	f.BoolVar(&c.Conf.MinusDiff, "minus-diff", false,
+		"Minus Difference between previous result and current result")
```

Comment:
It is easier to understand `-diff-plus` and `-diff-minus` next to each other when vuls report -help`.
Please change them as below.

```
  -diff-minus
        Minus Difference between previous result and current result
  -diff-plus
        Plus Difference between previous result and current result
```